### PR TITLE
Add Board and Task Features

### PR DESCRIPTION
**Pull request by durnasaban**
I added Board and Task Features

### CODE REVIEW

File: src/KoronaToDo/KoronaToDo.ConsoleApp/KoronaTask.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+        public User Assignee { get; private set; }
+
+        public DateTime DueDate { get; set; }
+
+        public void AssignToUser(User user)
+        {
+            Guard.Requires(user, nameof(User)).IsNotNull();
+
+            // Send Notification to old assignee
```
Comment (MevlutArslan) [inline src/KoronaToDo/KoronaToDo.ConsoleApp/KoronaTask.cs:27]: I would remove descriotions

File: src/KoronaToDo/KoronaToDo.ConsoleApp/Program.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            var user = new User();
+            user.FullName = "Korona Can";
+
+            task.AssignToUser(user);
+
+            Console.WriteLine($"Board Name : {board.Name}");
+
+            foreach (var koronaTask in board.KoronaTasks)
+            {
+                Console.WriteLine($"Task Name : {koronaTask.Name}");
```
Comment (MevlutArslan) [inline src/KoronaToDo/KoronaToDo.ConsoleApp/Program.cs:22]: These codes are not useful. 
I would move this logic to "ToString" method of the objects

File: src/KoronaToDo/KoronaToDo.ConsoleApp/KoronaTask.cs (hunk 1 above)
Comment (durnasaban) [reply src/KoronaToDo/KoronaToDo.ConsoleApp/KoronaTask.cs:27]: Done

File: src/KoronaToDo/KoronaToDo.ConsoleApp/Program.cs (hunk 2 above)
Comment (durnasaban) [reply src/KoronaToDo/KoronaToDo.ConsoleApp/Program.cs]: Done

**Pull request merged**